browser/speedreader/speedreader_browsertest.cc

```diff
@@ -47,6 +49,7 @@ class SpeedReaderBrowserTest : public InProcessBrowserTest {
 };
 
 IN_PROC_BROWSER_TEST_F(SpeedReaderBrowserTest, SmokeTest) {
```

Comment:
more than a smoke test would be good

Reply:
I think I'll add more test when a component updater stuff is in place